lib/tasks/daily_maintenance.rake

```diff
@@ -4,4 +4,5 @@ desc "Daily maintenance tasks"
 task daily_maintenance: :environment do
   ReadRibbon.expire_old_ribbons!
   CommentStat.daily_fill!
+  StatsGraphsJob.perform_now
```

Comment:
We don't want to `perform_now`, this should be enqueuing the job.

Reply:
I used `perform_later`, let me know if it looks right.

Reply:
Great!